src/cli/cli.go

```diff
@@ -278,6 +278,7 @@ func NewApp(cfg Config) (*App, error) {
 	app.CommandNotFound = func(ctx *gcli.Context, command string) {
 		tmp := fmt.Sprintf("{{.HelpName}}: '%s' is not a {{.HelpName}} command. See '{{.HelpName}} --help'.\n", command)
 		gcli.HelpPrinter(app.Writer, tmp, app)
+		gcli.OsExiter(1)
```

Comment:
Doesn't this make it always exit with 1?

Reply:
It's the same as `os.Exit(1)` here if the `OsExiter` is not set.